lib/experimental/blocks.php

```diff
@@ -159,14 +158,11 @@ function process_block_bindings( $block_content, $block, $block_instance ) {
 				}
 
 				// Process the HTML based on the block and the attribute.
-				$modified_block_content = block_bindings_replace_html( $modified_block_content, $block['blockName'], $binding_attribute, $source_value );
+				$modified_block_content = wp_block_bindings_replace_html( $modified_block_content, $block_instance->name, $binding_attribute, $source_value );
 			}
 			return $modified_block_content;
 		}
-
-		// Add filter only to the blocks in the list.
-		foreach ( $block_bindings_allowed_blocks as $block_name => $attributes ) {
-			add_filter( 'render_block_' . $block_name, 'process_block_bindings', 20, 3 );
-		}
 	}
+
+	add_filter( 'render_block', process_block_bindings, 20, 3 );
```

Comment:
This throws an error when I tested this:

```
Undefined constant "process_block_bindings"
```

Should this be:

```suggestion
	add_filter( 'render_block', 'process_block_bindings', 20, 3 );
```

instead?

Reply:
That definitely should be a string. In addition, it's going to be Gutenberg specific helper method so it should be prefixed with `gutenberg_`. In the WordPress core we will be able to tap into internal classes instead of using the hook.

Reply:
Ok I changed it to a string and renamed it in https://github.com/WordPress/gutenberg/pull/57742/commits/e6473a7af412a1a9d06d77e2309c38a1b02cc50a

So strange that I don't see the error on my machine though 🤔 Seems to work as intended regardless.

Anyway, it's been fixed 🙏